### ISSUE TITLE
What model did you use to generate the human segmentation that includes the neck label?

### ISSUE THREAD

**Issue by RubReh** [minar09/cp-vton-plus#14]

**Comment by thaithanhtuan** [comment]
>For better results, I would recommend using predicted segmentation by CIHP_PGN pre-trained network, which has a torso-skin label and produces better segmentation prediction.
https://github.com/minar09/cp-vton-plus/issues/3#issuecomment-660575377
You can run dataset_neck_skin_correction.py on VITON dataset or use above model as recommended by @minar09

**Comment by minar09** [comment]
> > For better results, I would recommend using predicted segmentation by CIHP_PGN pre-trained network, which has a torso-skin label and produces better segmentation prediction.
> > [#3 (comment)](https://github.com/minar09/cp-vton-plus/issues/3#issuecomment-660575377)
> > You can run dataset_neck_skin_correction.py on VITON dataset or use above model as recommended by @minar09

dataset_neck_skin_correction.py is just a heuristic method, which does not use any pretrained model. It just find the neck area on the existing segmentation and add the label. For more better training/testing, CIHP_PGN pre-trained model is better.

**Comment by minar09** [comment]
Closing the issue as its resolved. Feel free to reopen in case there are still problems.
